src/components/utils/YodaMeta.js

```diff
@@ -0,0 +1,38 @@
+class YodaMeta {
+
+  // Sign In type
+  // 1 : Facebook
+  // 2 : LinkedIn
+  static LOGIN_TYPE_FB = '1';
```

Comment:
Aren't we declaring something similar on the backend as well? 

https://github.com/sv-bootcamp/yoda/blob/master/src/server/models/users.model.js#L13

Is this becoming a source of duplicated constant declarations across backend and frontend? We should at least leave a TODO here to remove these dupes by merging the 2 repos and sharing common code like Goober does.  e.g. https://github.com/sv-bootcamp/goober/tree/master/src/shared